setup/test/integration/test_integration.py

```diff
@@ -99,6 +99,7 @@ def test_files_synced():
         .decode()
         .split("\n")
     )
+    print(changed_files)
```

Comment:
print needed? or is it better for test debug. can keep it

Reply:
Removed